crates/gitbutler-tauri/tauri.conf.nightly.json

```diff
@@ -27,8 +27,6 @@
 	},
 	"plugins": {
 		"updater": {
-			"active": true,
-			"dialog": false,
```

Comment:
Does this disable the updater for nightly builds?

Reply:
No these fields were just removed apparently, see: https://v2.tauri.app/start/migrate/from-tauri-1/#tauri-configuration